tests/components/zeroconf/test_init.py

```diff
@@ -34,7 +34,6 @@ def service_update_mock(zeroconf, service, handlers):
 
         assert await async_setup_component(
             hass, zeroconf.DOMAIN, {zeroconf.DOMAIN: {}})
-        await hass.async_block_till_done()
 
     assert len(MockServiceBrowser.mock_calls) == 2
-    assert len(mock_config_flow.mock_calls) == 2
+    assert len(mock_config_flow.mock_calls) == 4
```

Comment:
Should we just make this a `> 1`? Otherwise we have to update this for every integration that is migrated?

Reply:
Can't we just set the count to the number of flows that are defined in the generated file ?

Reply:
That works for me as well